core/state/state_object.go

```diff
@@ -351,7 +335,6 @@ func (s *stateObject) updateTrie(db Database) (Trie, error) {
 func (s *stateObject) updateRoot(db Database) {
 	tr, err := s.updateTrie(db)
 	if err != nil {
-		s.setError(fmt.Errorf("updateRoot (%x) error: %w", s.address, err))
```

Comment:
This operation is removed, since the returned error is already memorized in `s.updateTrie` function.